moveit_core/distance_field/include/moveit/distance_field/propagation_distance_field.h

```diff
@@ -443,30 +440,32 @@ class PropagationDistanceField : public DistanceField
   {
     return max_distance_sq_;
   }
+  typedef std::vector<Eigen::Vector3i, Eigen::aligned_allocator<Eigen::Vector3i>> StlVectorEigenVector3i;
 
 private:
-  typedef std::set<Eigen::Vector3i, compareEigen_Vector3i, Eigen::aligned_allocator<Eigen::Vector3i>> VoxelSet; /**< \brief Typedef for set of integer indices */
-  /**
-   * \brief Initializes the field, resetting the voxel grid and
-   * building a sqrt lookup table for efficiency based on
-   * max_distance_.
-   *
-   */
+  typedef std::set<Eigen::Vector3i, compareEigen_Vector3i, Eigen::aligned_allocator<Eigen::Vector3i>>
+      VoxelSet; /**< \brief Typedef for set of integer indices */
+                /**
```

Comment:
This formatting looks weird. Try to insert a new line after `VoxelSet` definition.